.editorconfig

```diff
@@ -0,0 +1,389 @@
+root = true
+
+[*]
+charset = utf-8
+end_of_line = lf
+indent_size = 2
+indent_style = space
+insert_final_newline = false
+max_line_length = 120
+tab_width = 2
+ij_continuation_indent_size = 8
```

Comment:
> From past experience we could think about using prettier or something similar in the future, just to avoid scenarios where our editors disagree.

Agreed. I use IJ WebStorm so this is for WS users primarily.